src/saturn_engine/worker_manager/api/jobs.py

```diff
@@ -93,3 +100,35 @@ def post_fetch_states() -> Json[FetchCursorsStatesResponse]:
             session=session,
         )
     return jsonify(FetchCursorsStatesResponse(cursors=cursors))
+
+
+@bp.route("/_start", methods=("POST",))
+def post_start_job() -> Json[JobsStartResponse]:
+    restart: bool = flask.request.args.get("_restart", "false") == "true"
+    start_input = marshall_request(StartJobInput)
+    if not start_input.name and not start_input.job_definition_name:
+        abort(http_code=400, error_code="MUST_SPECIFY_JOB")
+
+    with session_scope() as session:
+        job: Job | None = None
+        if start_input.name:
+            job = jobs_store.get_job(session=session, name=start_input.name)
+        elif start_input.job_definition_name:
+            job = jobs_store.get_last_job(
+                session=session, job_definition_name=start_input.job_definition_name
+            )
+
+        if not job:
+            abort(http_code=404, error_code="JOB_NOT_FOUND")
+
+        if restart:
+            jobs_store.update_job(session=session, name=job.name, completed_at=utcnow())
```

Comment:
if `not restart` and an existing job is running then we should not create a new job?

Reply:
I thought the goal was if restart is false, we just start a new job without looking if there's one already scheduled/running. If restart is true, we look for a currently scheduled/running and stop it before creating a new one.